plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/XProjectManager.java

```diff
@@ -84,6 +84,16 @@
 	@Inject
 	protected OperationCanceledManager operationCanceledManager;
 
+	/**
+	 * The map for this project's resource set.
+	 */
+	@Inject
+	protected UriResourceMap uriResourceMap;
+
+	/** The workspace manager */
+	@Inject
+	protected XWorkspaceManager workspaceManager;
```

Comment:
This field seems to be not used in this class.

Reply:
It is used in org.eclipse.n4js.ide.xtext.server.WorkspaceAwareResourceLocator

Reply:
Refactored